content/applications/websites/ecommerce/checkout_payment_shipping/ewallet_giftcard.rst

```diff
@@ -0,0 +1,122 @@
+======================
+eWallet and gift cards
+======================
+
+You can enable **eWallets** and **gift cards** for your customers on your eCommerce, which they can
+use to for their online and in-store shopping. To allow eWallets and gift cards for your eCommerce,
+head to :menuselection:`Website app --> eCommerce` and in the :guilabel:`Shop - Products` category,
+enable :guilabel:`Discounts, Loyalty & Gift Card`.
```

Comment:
I don't see the "shop-products" category 
![image](https://user-images.githubusercontent.com/85342080/204994193-8c2eb418-f2e6-4d38-8b88-1b2893b0cd72.png)

Reply:
My bad, it was the wrong menu :)